modules/openapi-generator/src/main/resources/Java/libraries/okhttp-gson/auth/OAuthOkHttpClient.mustache

```diff
@@ -1,5 +1,4 @@
-{{#hasOAuthMethods}}
-package {{invokerPackage}}.auth;
+package com.bosch.pace.io.azure.functions.models.auth;
```

Comment:
I think this change needs to be reverted. please have a look.

Reply:
Totally right 😅